src/client/pages/SignIn.stories.tsx

```diff
@@ -13,3 +13,22 @@ export const Default = () => <SignIn />;
 Default.story = {
   name: 'with defaults',
 };
+
+export const withEmail = () => <SignIn email="test@example.com" />;
```

Comment:
Not sure if this makes much difference but I've always followed the convention of functions that return jsx having a leading capital letter - to mark them as components